utils/logger.go

```diff
@@ -0,0 +1,15 @@
+package utils
+
+import (
+	"log"
+)
+
+var (
+	DebugMode = true
+)
+
+func Debug(format string, v ...interface{}) {
+	if DebugMode {
+		log.Printf("[DEBUG] "+format, v...)
```

Comment:
Why not use klog directly in the user functions? `gin.Context` implements `context.Context`.